### PR TITLE
Todo delete

### PR DESCRIPTION
**Pull request by muskaanshah**
User can delete todos which are not required

### CODE REVIEW

**Review by rutvikpumak — APPROVED** [review]
Code looks clean and well structured
Delete functionality working as expected 
Good to merge

File: src/reducers/todoreducer.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ const todoCompletedHandler = (state, id) => {
...
 };
 
+const clearTodoHandler = (state, id) => {
+    const temp = state.todo.filter(curTodo => curTodo.id !== id);
+    return { ...state, todo: temp };
+}
+
 const todoReducer = (state, action) => {
     switch (action.type) {
         case "START_TODO_LIST":
```
Comment (znjs) [inline src/reducers/todoreducer.js:21]: Use variable instead of strings to avoid typos

File: src/reducers/todoreducer.js [hunk 2]
```diff
@@ -29,6 +34,8 @@ const todoReducer = (state, action) => {
             return { ...state, todo: action.payload.value };
         case "SET_TODO_COMPLETED":
             return todoCompletedHandler(state, action.payload.value);
+        case "CLEAR_TODO":
```
Comment (znjs) [inline src/reducers/todoreducer.js:37]: Use variable instead of strings to avoid typos

File: src/reducers/todoreducer.js [hunk 3]
```diff
@@ -29,6 +34,8 @@ const todoReducer = (state, action) => {
             return { ...state, todo: action.payload.value };
         case "SET_TODO_COMPLETED":
```
Comment (znjs) [inline src/reducers/todoreducer.js:35]: Use variable instead of strings to avoid typos

**Review by znjs — APPROVED** [review]
Good namings used
Good folder structure

**Pull request merged**